<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormSystemOfNonLinearEquations</class>
 <widget class="QWidget" name="FormSystemOfNonLinearEquations">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>608</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Решение систем нелинейных уравнений</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0" colspan="2">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Уравнения системы</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2" colspan="2">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Значения переменных</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QTableView" name="tableViewEquations">
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>300</number>
         </attribute>
        </widget>
       </item>
       <item row="1" column="2" colspan="2">
        <widget class="QTableView" name="tableViewVarValues">
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>130</number>
         </attribute>
        </widget>
       </item>
       <item row="2" column="0">
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Количество
уравнений</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="spinBoxNumberOfEquations"/>
         </item>
        </layout>
       </item>
       <item row="2" column="1" colspan="2">
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Количество
десятичных знаков</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="spinBoxPrec"/>
         </item>
        </layout>
       </item>
       <item row="2" column="3">
        <layout class="QFormLayout" name="formLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Заданная
погрешность</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QDoubleSpinBox" name="doubleSpinBoxEps"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Решение</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" rowspan="2">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Ход выполнения 
итеррационного процесса</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" rowspan="2">
      <widget class="QTableView" name="tableViewResult">
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>130</number>
       </attribute>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QTextEdit" name="textEditIterProcess">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <layout class="QFormLayout" name="formLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Количество
итерраций</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEditNumberOfIterrations"/>
       </item>
      </layout>
     </item>
     <item row="4" column="1">
      <layout class="QFormLayout" name="formLayout_5">
       <item row="0" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Среднеквадратическая
невязка</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEditMeanSquareResidual">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="pushButtonCalculate">
       <property name="text">
        <string>Найти решение</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="pushButtonClose">
       <property name="text">
        <string>Выйти</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
